preload/avoid_delaying_onload_link_preload.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+    var t = async_test('Makes sure link preload preloaded resources are not delaying onload');
+</script>
+<link rel=preload href="resources/dummy.js?pipe=trickle(d1)" as=script>
```

Comment:
This seems like a possible source of unstable results. Maybe something like 5 seconds? @jgraham